projects/plugins/wpcomsh/wpcom-features/functions-wpcom-features.php

```diff
@@ -189,7 +189,7 @@ function _wpcom_features_get_simple_site_purchases( $blog_id ) {
 	$purchases = $wpdb->get_results(
 		$wpdb->prepare(
 			"
-					SELECT 
+					SELECT
```

Comment:
Don't think this change is needed?

Reply:
Not needed yes, the editor automatically remove it 🙂